src/Proto.Cluster/Identity/IdentityStorageWorker.cs

```diff
@@ -170,6 +179,10 @@ await _storage.WaitForActivationAsync(clusterIdentity, ct)
             }
             catch (Exception e)
```

Comment:
Were we going to add retries to GetWithGlobalLock? Better to retry a few times here than failing all waiting request and retrying those?